codecov_cli/helpers/options.py

```diff
@@ -0,0 +1,52 @@
+import click
+
+from codecov_cli.fallbacks import CodecovOption, FallbackFieldEnum
+from codecov_cli.helpers.git import GitService
+
+_global_options = [
+    click.option(
+        "-C",
+        "--sha",
+        "--commit-sha",
+        "commit_sha",
+        help="Commit SHA (with 40 chars)",
+        cls=CodecovOption,
+        fallback_field=FallbackFieldEnum.commit_sha,
+        required=True,
+    ),
+    click.option(
+        "-Z",
+        "--fail-on-error",
+        "fail_on_error",
+        is_flag=True,
+        help="Exit with non-zero code in case of error",
+    ),
+    click.option(
+        "--git-service",
+        cls=CodecovOption,
+        fallback_field=FallbackFieldEnum.git_service,
+        type=click.Choice(service.value for service in GitService),
```

Comment:
please change this to `click.Choice([service.value for service in GitService])` otherwise some tests will fail ATS.

Why? I don't know exactly, but if you look on the test `test_upload_process_options` you can see that the list is empty. However if you run the command locally the list will not be empty. But wrapping it in a list first will solve the issue.